test/loopback.test.js

```diff
@@ -608,4 +611,96 @@ describe('loopback', function() {
       expect(methodNames).to.include('prototype.instanceMethod');
     });
   });
+
+  describe('Remote method inheritance', function() {
+    function getAllMethodNamesWithoutClassName(Model) {
```

Comment:
We usually put helper methods at the end of the describe block, see https://github.com/strongloop/loopback-contributor-docs/blob/671a797623450fa94f5e3da05d12c34fc8cd5aa1/style-guide.md#layout-of-test-files. It mentions before/after hooks only, but the same principle applies to helper methods. Could you send a PR to clarify that?